cross/python3/Makefile

```diff
@@ -24,6 +24,13 @@ ADDITIONAL_CFLAGS = -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -L $(STAGING_IN
 CONFIGURE_ARGS  = --enable-shared --enable-ipv6 --without-ensurepip --enable-loadable-sqlite-extensions PGEN_FOR_BUILD=$(HOSTPGEN)
 CONFIGURE_ARGS += ac_cv_buggy_getaddrinfo=no ac_cv_file__dev_ptmx=no ac_cv_file__dev_ptc=no ac_cv_have_long_long_format=yes
 
+ifeq ($(findstring $(ARCH), qoriq),$(ARCH))
```

Comment:
At this stage the point of this was to, at least, fix qoriq version.

I would need a little bit more digging to understand which platforms are actually big endian and, maybe, at this stage it would be wise to explicitely define this hack at the level of the TC since, then, it would fix the usage of the standard `AC_C_BIGENDIAN` macro of autoconf which seems to not work in our cases :( But then the root cause may also be an invalid toolchain configuration.